src/test/java/com/exasol/dbbuilder/dialects/mysql/MySQLDatabaseObjectCreationIT.java

```diff
@@ -0,0 +1,158 @@
+package com.exasol.dbbuilder.dialects.mysql;
+
+import static com.exasol.dbbuilder.dialects.mysql.MySqlGlobalPrivilege.CREATE_ROLE;
+import static com.exasol.dbbuilder.dialects.mysql.MySqlGlobalPrivilege.CREATE_USER;
+import static com.exasol.dbbuilder.dialects.mysql.MySqlObjectPrivilege.DELETE;
+import static com.exasol.dbbuilder.dialects.mysql.MySqlObjectPrivilege.SELECT;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import java.sql.*;
+
+import org.junit.jupiter.api.*;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.dbbuilder.dialects.*;
+
+@Tag("integration")
+@Testcontainers
+class MySQLDatabaseObjectCreationIT {
+    private static final String MYSQL_DOCKER_IMAGE_REFERENCE = "mysql:8.0.20";
+    @Container
+    private static final MySQLContainer<?> container = new MySQLContainer<>(MYSQL_DOCKER_IMAGE_REFERENCE)
+            .withUsername("root").withPassword("");
+    private MySqlObjectFactory factory;
+    private Connection adminConnection;
+
+    @BeforeEach
+    void beforeEach() throws SQLException {
+        this.adminConnection = container.createConnection("");
+        this.factory = new MySqlObjectFactory(container.createConnection(""));
+    }
+
+    @Test
+    void testCreateSchema() {
+        final Schema schema = this.factory.createSchema("THE_SCHEMA");
+        assertSchemaExistsInDatabase(schema.getName());
+    }
+
+    private void assertSchemaExistsInDatabase(final String schemaName) {
```

Comment:
Which new part of the production code is that test supposed to exercise? I looks like it tests the assert.

Reply:
I didn't get the question. It's the same thing as `assertObjectExistsInDatabase` in Exasol, but the objects information stored in different places in MySQL, so I have a few assert methods